packages/docs-app/src/examples/datetime-examples/dateRangePickerExample.tsx

```diff
@@ -119,6 +120,7 @@ export class DateRangePickerExample extends React.PureComponent<IExampleProps, I
                     />
                 </div>
                 <div>
+                    <H5 />
```

Comment:
??

Reply:
it looks off without it, but yes it's gross so I'll remove it